yt_dlp/postprocessor/ffmpeg.py

```diff
@@ -747,8 +747,8 @@ def add(meta_list, info_list=None):
         add('track', 'track_number')
         add('artist', ('artist', 'artists', 'creator', 'creators', 'uploader', 'uploader_id'))
         add('composer', ('composer', 'composers'))
-        add('genre', ('genre', 'genres'))
-        add('album')
+        add('genre', ('genre', 'genres', 'tags', 'categories'))
```

Comment:
IMO `categories` will usually be more desirable than `tags` since `tags` can be hashtags and search keywords, etc
```suggestion
        add('genre', ('genre', 'genres', 'categories', 'tags'))
```